src/components/scoreboard/status-indicator/StatusIndicator.jsx

```diff
@@ -9,18 +9,19 @@ const style = {
 };
 
 function getIcon(code) {
-  return code ? "error" : "warning";
+  return "error";
```

Comment:
Does engine only ever return errors and not warnings? Just curious since we'll now only be able to display the error ❗icon and never show the warning ⚠️ icon.